.github/workflows/test.yaml

```diff
@@ -38,10 +39,71 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
-        if: always()
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Save Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Pull a small test Docker image.
+        run: docker pull ghcr.io/linuxserver/baseimage-alpine:edge-version-f502fcfd
+  restore-cache:
+    name: Restore Cache
+    needs:
+      - save-cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Load Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Verify Docker loaded empty image from cache.
+        run: |
+          description="$(docker inspect --format '{{ index .Config.Labels "description" }}' empty)"
+          [[ $description == "empty image" ]]
+  delete-cache:
+    name: Delete Cache
+    if: needs.save-cache.result == 'success'
+    needs:
+      - save-cache
+      - restore-cache
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: write
+    steps:
+      - name: Delete existing test cache.
```

Comment:
nit: Remove "existing" here since it can be read to imply that we are about to create some other test cache.